docs/user-guide/sns-instruments/POWGEN_data_reduction.ipynb

```diff
@@ -386,7 +407,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.12"
+   "version": "3.10.13"
   }
  },
  "nbformat": 4,
```

Comment:
I'm not sure I see peak fitting or removal used in the notebook? Am I missing something?